<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Garamond&quot;;
background-color: rgb(30, 58, 138);
background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../photos/paladin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewPasswordButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border-radius: 40px;  /* Połowa szerokości/wysokości */
    font-size: 18px;
    width: 80px;  
    height: 80px;  
	color:rgb(255, 215, 0);
	background-color: rgb(44, 44, 44);

}
QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1c6ea4;
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border-radius: 40px;  /* Połowa szerokości/wysokości */
    font-size: 18px;
    width: 80px;  
    height: 80px;  
	color:rgb(255, 215, 0);
	background-color: rgb(44, 44, 44);

}
QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #1c6ea4;
}</string>
   </property>
   <property name="text">
    <string>RETURN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeatPasswordTextbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTextbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginTextbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameTextbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
